Remotion/ObjectBinding/UnitTests/BusinessObjectReferenceDataSourceBaseTests/SaveValue.cs

```diff
@@ -93,26 +94,26 @@ public void ClearsHasBusinessObjectCreated ()
     [Test]
     public void ParentIsNull_DoesNotSaveValueIntoBoundObject_AndReturnsFalse ()
     {
-      var parentObjectStub = _referencedDataSourceStub.BusinessObject;
-      _referencedDataSourceStub.BusinessObject = null;
-      var expectedValue = MockRepository.GenerateStub<IBusinessObject> ();
+      var parentObjectStub = _referencedDataSourceStub.Object.BusinessObject;
+      _referencedDataSourceStub.Object.BusinessObject = null;
+      var expectedValue = new Mock<IBusinessObject>();
 
-      var referenceDataSource = new TestableBusinessObjectReferenceDataSource (_referencedDataSourceStub, _referencePropertyStub);
-      referenceDataSource.BusinessObject = expectedValue;
+      var referenceDataSource = new TestableBusinessObjectReferenceDataSource (_referencedDataSourceStub.Object, _referencePropertyStub.Object);
+      referenceDataSource.BusinessObject = expectedValue.Object;
 
       var result = referenceDataSource.SaveValue (false);
 
       Assert.That (result, Is.False);
-      parentObjectStub.AssertWasNotCalled (stub => stub.SetProperty (null, null), options=>options.IgnoreArguments());
+      Mock.Get (parentObjectStub).Verify (stub => stub.SetProperty (It.IsAny<IBusinessObjectProperty>(), It.IsAny<object>()), Times.Never());
```

Comment:
The manual update of the IgnoreArguments, is this already covered by the tooling or still manual?

Reply:
`IgnoreArguments` can be converted automatically. This is a special case which cannot be converted automatically.
e.g.
```
// this can be converted automatically
parentObjectStub.Stub (_ => _.SetProperty (null, null)).IgnoreArguments();
// must be converted manually
parentObjectStub.AssertWasNotCalled (stub => stub.SetProperty (null, null), options=>options.IgnoreArguments());
```

Reply:
Hmm.. can we force a compiler error at this point during the migration, perhaps replacing `null` with `/*null*/`? Because passing null and expecting this is never called will always be green and invalidate the assertion's intention.